dash/development/_jl_components_generation.py

```diff
@@ -23,59 +26,48 @@
 export {funcname}
 
 """
-    {funcname}(;kwargs...)
-    {funcname}(children::Any;kwargs...)
-    {funcname}(children_maker::Function;kwargs...)
+    {funcname}(;kwargs...){childrens_signatures}
```

Comment:
children is already plural, let's just call this `children_signatures` (also `children_definitions`)

Reply:
@alexcjohnson fixed in 650b9d3